ERCS/erc-draft_composite_policy.md

```diff
@@ -0,0 +1,425 @@
+---
+title: Universal Policy Engine
```

Comment:
Would "Composite policy engine" be a more descriptive title?  Universal doesn't seem to add much.  

I am not an ERC editor